README.md

```diff
@@ -105,271 +123,137 @@ Follow these steps to configure your access token and ingest endpoint:
      SIGNALFX_ENDPOINT_URL=https://ingest.<realm>.signalfx.com
     ```
 
-If you're sending data to an OpenTelemetry Collector, you have to specify the full path like this: `http://<otel-collector-host>:9411/api/v2/spans`
+   If you're sending data to an OpenTelemetry Collector, you have to specify
+   the full path like this: `http://<otel-collector-host>:9411/api/v2/spans`
 
-3. (Optional) Globally enable/disable metrics and/or tracing by setting SIGNALFX_TRACING_ENABLED and/or SIGNALFX_METRICS_ENABLED.
+3. (Optional) Globally enable metrics by setting `SIGNALFX_METRICS_ENABLED`:
     ```bash
-    SIGNALFX_TRACING_ENABLED=true  [defaults to true]
     SIGNALFX_METRICS_ENABLED=false [defaults to false]
     ```
 
-4. (Optional) Enable context propagation (currently only supports B3 propagation).
+4. (Optional) Enable context propagation. The wrapper currently supports only
+   B3 context propagation.
     ```bash
     SIGNALFX_CTX_PROPAGATION_ENABLED=true [defaults to true]
     ```
 
 5. (Optional) Specify other environment variables to better configure the traces.
-For a list of all the available configuration options available, see
-[Configure the SignalFx Tracing Library for .NET](https://github.com/signalfx/signalfx-dotnet-tracing#configure-the-signalfx-tracing-library-for-net). 
 
-## (Optional) Step 4: Reduce the size of deployment packages with AWS Lambda layers
+   For a list of all the available configuration options available, see
+   [Configure the SignalFx Tracing Library for .NET](https://github.com/signalfx/signalfx-dotnet-tracing#configure-the-signalfx-tracing-library-for-net). 
+
+## Deploy the Lambda wrapper with a Lambda layer
 
 Add a layer that includes the SignalFx Lambda wrapper to your Lambda function.
```

Comment:
I think that the bit reducing the deployment size can be used here, something along the lines: To reduce the deployment size ... and so on.